docs/tutorial/fuses.md

```diff
@@ -0,0 +1,52 @@
+# Electron Fuses
+
+> Package time feature toggles
+
+## What are fuses?
+
+For a subset of Electron functionality it makes sense to disable certain features for an entire application.  For example, 99% of apps don't make use of `ELECTRON_RUN_AS_NODE`, these applications want to be able to ship a binary that is incapable of using that feature.  We also don't want Electron consumers building Electron from source as that is both a massive technical challenge and has a high cost of both time and money.
+
+Fuses are the solution to this problem, at a high level they are "magic bits" in the Electron binary that can be flipped when packaging your Electron app to enable / disable certain features / restrictions.  Because they are flipped at package time before you code sign your app the OS becomes responsible for ensuring those bits aren't flipped back via OS level code signing validation (Gatekeeper / App Locker).
+
+## How do I flip the fuses?
+
+### The easy way
+
+We've made a handy module `@electron/fuses` to make flipping these fuses easy.  Check out the README of that module for more details on usage and potential error cases.
```

Comment:
Does this exist yet? We probably shouldn't commit this unless this exists 🤣

Reply:
It does but I wanted to test it against a real Electron release before `npm publish`'ing 😆

Reply:
I guess this won't show up on the website until Electron 12 becomes stable so go ahead :)